### ISSUE TITLE
[Suggestion] Nitor-Based Golems/Jinn

### ISSUE THREAD

**Issue by praecipitator** [Azanor/thaumcraft-suggestions#257]
Cleaned up copypaste from what I posted in the forum.

"According to an old legend, the Jinn were created from smokeless fire. Considering that Nitor is smokeless fire, this gives you an idea".

Basically, you craft a magic lamp, place it down, and it spawns a special kind of golem.

The lamp is crafted from alchemical brass and nitor, and needs to be infused to create the jinn.
The jinn can only act in a certain area around the lamp. It cannot be interacted with, but the lamp can have a color assigned, and be picked up and placed somewhere else. 

The lamp would also be what recieves upgrades, like the aggression module, the brain, the eyes. In this case, the different types of hands could also be upgrades, which you need to apply to the lamp.

If the jinn is killed, the lamp reverts back to being just a lamp. It might or might not keep the upgrades.

In the original Aladdin story, there was also a second, weaker jinn in a ring. That might be an interesting bauble. I can think of two possibilities:

- Infuse a ring with spiritus, humanus, nitor and a guard sigil. In combat, the ring sometimes spawns a weak jinn for a few seconds which can shoot a few fireballs.

- Instead of the guard sigil, use a gather sigil. If there are items lying around you, a jinn will be spawned which picks them up and bring them to you

**Comment by angeltxilon** [comment]
Yes. Djinns gollemancy could be a very great addition for thaumcraft!
